docs/tutorials/machine_learning.ipynb

```diff
@@ -86,6 +104,312 @@
     "\n",
     "fitted_model.predict(dataset=test_set)\n"
    ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "start_time": "2023-06-23T08:54:28.938660Z",
+     "end_time": "2023-06-23T08:54:28.956511Z"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Metrics\n",
+    "\n",
+    "A machine learning metric, also known as an evaluation metric, is a measure used to assess the performance of a machine learning model on a test set or is used during cross-validation to gain insights about performance and compare different models or parameter settings.\n",
+    "In `Safe-DS`, the available metrics are: `Accuracy`, `Confusion Matrix`, `F1-Score`, `Precision`, and `Recall`. Before we go through each of these in detail, we need an understanding of the different `components of evaluation metrics`.\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Components of evaluation metrics\n",
+    "\n",
+    "These are distinct elements or parts that contribute to the overall assessment of an evaluation measure.\n",
+    "* `True positives` TP: the positive tuples that the classifier correctly labeled.\n",
+    "* `False positives` FP : the negative tuples that were falsely labeled as positive.\n",
+    "* `True negatives` TN: the negative tuples that the classifier correctly labeled.\n",
+    "* `False negatives` FN: the positive tuples that were falsely labeled as negative.\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Confusion Matrix\n",
+    "A confusion matrix is a table that is used to define the performance of a classification algorithm. It classifies the predictions to be either true positive, true negative, false positive or false negative. It has no Formula.\n",
```

Comment:
```suggestion
    "A confusion matrix is a table that is used to define the performance of a classification algorithm. It classifies the predictions to be either true positive, true negative, false positive or false negative.\n",
```

Reply:
I'll prefer to leave it that way for consistency with other metrics. Because it's a metric, some users might think there is a special formula needed to derive a confusion matrix.